apps/condo/domains/user/oidc/configuration.js

```diff
@@ -8,6 +8,11 @@ const HTTPS_REGEXP = /^https:/
 
 module.exports = function createConfiguration (context, conf) {
     const jwksStr = get(conf, 'JWKS')
+    const cookieKeys = JSON.parse(get(conf, 'COOKIE_KEYS', '[]'))
+
+    if (isEmpty(cookieKeys)) {
```

Comment:
Do we need to check that it is an array?

Reply:
Done